server/schedule/operator/operator.go

```diff
@@ -125,6 +125,16 @@ func (o *Operator) Kind() OpKind {
 	return o.kind
 }
 
+// SchedulerKind return the highest OpKind even if the operator has many OpKind
+// fix #3778
+func (o *Operator) SchedulerKind() OpKind {
+	// LowBit ref: https://zh.wikipedia.org/wiki/%E6%A0%91%E7%8A%B6%E6%95%B0%E7%BB%84
```

Comment:
This is a Chinese page. You can use `https://en.wikipedia.org/wiki/Find_first_set`.